app/Http/Controllers/AuthController.php

```diff
@@ -65,6 +72,46 @@ public function signOut()
     {
         Session::flush();
         Auth::logout();
-        return Redirect('login');
+        return Redirect('home');
```

Comment:
`return Redirect('home');`  change to `return redirect('home');`

Remove the `Session::flush();`  if it is not needed